src/app/editor/editor.js

```diff
@@ -53,6 +63,13 @@ function Editor (opts = {}) {
   var self = this
   var el = yo`<div id="input"></div>`
   var editor = ace.edit(el)
+  // editor.setTheme('ace/theme/solarized_dark')
+  // editor.setTheme('ace/theme/clouds_midnight')
+  // editor.setTheme('ace/theme/pastel_on_dark')
+  // editor.setTheme('ace/theme/tomorrow_night')
+  editor.setTheme('ace/theme/tomorrow_night_blue')
```

Comment:
see above: 
var styleguide = require('remix-lib').ui.styleGuide
if (styleguide.aceTheme) {
editor.setTheme(styleguide.aceTheme) ==> where styleguide.aceTheme is 'ace/theme/tomorrow_night_blue' 
}